### ISSUE TITLE
how to do global customize to jupyter notebook in ubuntu

### ISSUE THREAD

**Issue by gcwenken** [jupyter/help#107]
Hi:

If I want to customize js or css, I should place them in ~/.jupyter/custom/custom.js and ~/.jupyter/custom/custom.css. However, I want to know is there any global setting to that? I mean is there any corresponding folder in /etc/jupyter/, /usr/local/etc/jupyter/, /usr/local/share/jupyter/, /usr/share/jupyter/ or any other directory not in the home folder. So the custom js and css are suitable for all user.

**Comment by takluyver** [comment]
I don't believe there is a system location for the `custom` files. You can provide custom nbextensions from a systemwide location (e.g. `/usr/local/share/jupyter/nbextensions`), but it would need a bit of per-user configuration to enable your extension.

**Comment by gcwenken** [comment]
Because I want to use mediawiki-texvc.js in mathjax, so I put:


MathJax.Hub.Config({TeX: {extensions: ["mediawiki-texvc.js"]}});


in the ~./jupyter/custom/custom.js


Then it works. For example, if I put $\euro$ in the markdown cell it will show €.


However, if I put the following


define(function()
    {
      function load_ipython_extension()
      {
        MathJax.Hub.Config({TeX: {extensions: ["mediawiki-texvc.js"]}});
      }
      return {load_ipython_extension: load_ipython_extension};
    });


into my.js and place my.js in /usr/share/jupyter/nbextensions and then use jupyter nbextension enable to enable my.js. It does not work as expected. If I put $\euro$ in the markdown cell it will show \euro. So how to correctly write the extension?

________________________________
